vector/src/main/java/im/vector/adapters/VectorParticipantsAdapter.java

```diff
@@ -1083,16 +1099,16 @@ public void onSuccess(Void info) {
             }
         }
 
-        statusTextView.setText("");
+        emailTextView.setText("");
         if (mContactsFilter == VectorRoomInviteMembersActivity.ContactsFilter.NO_TCHAP_ONLY) {
             // Use the status text view to display the contact email if any
             if (participant.mContact.getEmails().size() > 0) {
-                statusTextView.setText(participant.mContact.getEmails().get(0));
+                emailTextView.setText(participant.mContact.getEmails().get(0));
             } else if (participant.mContact.getPhonenumbers().size() > 0) {
-                statusTextView.setText(participant.mContact.getPhonenumbers().get(0).mRawPhoneNumber);
+                emailTextView.setText(participant.mContact.getPhonenumbers().get(0).mRawPhoneNumber);
             }
         } else {
-            statusTextView.setText(status);
+            emailTextView.setText(status);
```

Comment:
We should not display detailed presence anymore even for mKnownContactsSectionPosition
The green light would be enough 
The param "status" can be removed from the code